### ISSUE TITLE
Login user flow

### ISSUE THREAD

**Issue by apotdevin** [byteleaf/companyon-frontend#41]
Currently the user login consists of:
1. Go to Companyon url
2. Get automatically redirected to auth page
3. Redirected back to Companyon url

Think it makes for a better UX if it follows something more along the lines of:
1. Go to Companyon url
2. If not authenticated go to a Companyon login page with a Login button
3. On button click then direct user to the auth page
4. Redirect back to Companyon url

But just an idea :)

**Comment by heerma** [comment]
Yep think so too. So the user gets at least some context for what he currently registers/logs in :D
